src/FeatureFlagsServiceProvider.php

```diff
@@ -68,18 +77,16 @@ function (Container $app) {
             }
         );
 
-        $this->app->singleton(
-            FeatureFlagUserConvertor::class,
+        $this->app->singleton(FeatureFlagOverrider::class,
             function (Container $app) {
-                /** @var Application $app */
-                $config = $app['config'];
+                /** @var ConfigRepository $config */
+                $config = $app->get('config');
 
                 /** @var string */
-                $convertor = $config->get('feature-flags.convertor');
-                return $app->get($convertor);
-            }
-        );
+                $convertor = $config->get('feature-flags.overrider');
```

Comment:
Think the only thing I'd change is to add the following above this instead of `/** @var string */` (mainly because PhpStorm is complaining about the current comment. But also because it adds further context.
```php
/** @var class-string<FeatureFlagOverrider> $convertor */
```

Reply:
Whoops... didn't realise I'd already marked it as approved. Double review. 😬 🤷🏻

Reply:
Ah ys, missed this one and not the other, thanks for spotting it 👍